pkg/minikube/bootstrapper/images/images.go

```diff
@@ -64,14 +76,54 @@ func componentImage(name string, v semver.Version, mirror string) string {
 	return fmt.Sprintf("%s:v%s", path.Join(kubernetesRepo(mirror), name), v)
 }
 
+// fixes 13136 by getting the latest image version from the k8s.gcr.io repository instead of hardcoded
+func findLatestTagFromRepository(url string, lastKnownGood string) string {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", url, nil)
+
+	if err != nil {
+		klog.Infof("Failed to get latest image version for %s, reverting to version %s\n", url, lastKnownGood)
```

Comment:
Since this is somewhat of a failure, I wonder if it would make more sense to make this `klog.Warningf`, what are your thought?

Reply:
Agreed, and fixed

Reply:
+1